packages/devtools-reps/src/reps/string.js

```diff
@@ -133,6 +149,23 @@ function getElementConfig(opts) {
   return config;
 }
 
+function maybeCropString(opts, text) {
+  const {
+    isLong,
+    shouldCrop,
+    cropLimit,
+  } = opts;
+
+  // Cropping of LongString has been handled before formatting.
```

Comment:
@nchevobbe I broke cropping into 2 functions for each string type (that use the same precalculated options), to emphasize that they are different from each other.